po/zh-Hant.po

```diff
@@ -2,7 +2,7 @@ msgid ""
 msgstr ""
 "Project-Id-Version: Comprehensive Rust 🦀\n"
 "POT-Creation-Date: \n"
-"PO-Revision-Date: 2023-06-01 22:43+0800\n"
+"PO-Revision-Date: 2023-06-02 14:43+0800\n"
```

Comment:
A small detail: I've configured my own editor to stop updating this field. It is from a forgotten era where people liked to add such identifiers inline — now we can use our version control system instead :smile:

Reply:
I see. I'll stop uploading this.